src/app/docs/select-field/select-field-docs.component.html

```diff
@@ -4,7 +4,20 @@
   pageTitle="Select field"
 >
   <sky-docs-demo-page-summary>
-    A select field creates a text input or button to launch a modal picker for users to select items from a searchable, browsable, and filterable list.
+    <p>
+      A select field creates a text input or button to launch a modal picker for users to select items from a searchable, browsable, and filterable list.
+    </p>
+  
+  <sky-alert alertType="warning">
+      We recommend the
+      <a *skyThemeIf="'default'" href="https://developer.blackbaud.com/skyux-lookup/docs/lookup">
+        lookup component
+      </a>
+      <a *skyThemeIf="'modern'" href="https://developer.blackbaud.com/skyux-lookup/docs/lookup?svcid=modern">
+        lookup component
+      </a>
+      instead of select field. We will deprecate the select field component in the next major version of SKY UX.
```

Comment:
Are we adding a deprecation warning in 5, or actually removing the component from the codebase? I wonder if this wording will cause panic one way or the other.

Reply:
I don't know the answer to removing vs. deprecating, but the wording here mostly follows the pattern in the definition list note at https://developer.blackbaud.com/skyux-layout/docs/definition-list?docs-active-tab=design. The thing it's missing though is the "because" clause to explain the rationale. Do you think adding that would help to ameliorate any panic?
 ```suggestion
      instead of the select field because we improved the lookup component to better handle the use cases of both components. We will deprecate the select field component in the next major version of SKY UX.
```